### ISSUE TITLE
NTR, pulmo: Atoll sign

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5582]
**Preferred term label:**
Atoll sign

**Synonyms**
Reversed halo sign

**Definition (free text, please give PubMed ID)**
CT finding of central ground-glass opacity surrounded by denser consolidation of crescentic shape (forming more than three-fourths of a circle) or complete ring of at least 2 mm in thicknes. A rare sign, it was initially reported to be specific for cryptogenic organizing pneumonia, but was subsequently described in patients with paracoccidioidomycosis 

PMID: 22553298, PMID: 10414267
https://www.ajronline.org/doi/full/10.2214/ajr.184.6.01841932

**Parent term (use hpo.jax.org/app)**
Ground-glass opacification on pulmonary HRCT HP:0025179

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
paracoccidioidomycosis (ORPHA:73260)
cryptogenic organizing pneumonia (ORPHA:1302 )

**Your nano-attribution (ORCID)**
LMU: crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
adding as HP:0032983
